platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -475,9 +477,13 @@ MGL_EXPORT
 /**
  Offset distance of icon from its anchor.
  
+ This property is measured in points multiplied by the value of "icon-size"s.
+ 
  The default value of this property is an `MGLStyleValue` object containing an
- `NSValue` object containing a `CGVector` struct set to 0 rightward and 0
- downward. Set this property to `nil` to reset it to the default value.
+ `NSValue` object containing a `CGVector` struct set to 0 points multiplied by
+ the value of "icon-size"s rightward and 0 points multiplied by the value of
```

Comment:
This is why sticking a phrase inside `units` is problematic. I guess we can special-case this property to omit the unit (even though `CGVector` normally specifies two point values).

Reply:
👍  I'm going to revert https://github.com/mapbox/mapbox-gl-js/commit/67bce60ecdf54e22702e1210b84dc819f42cea5b and explain this in the prose instead.

Reply:
OK. For what it’s worth, 5374da28f586bf82ea2df60572b3821dc7940dca adds support for resolving property references in backticks and also omits unwieldy units from struct descriptions. No harm in merging mapbox/mapbox-gl-js#5958 though.